.devcontainer/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# See here for image contents: https://github.com/NYCPlanning/docker-geosupport
+
+# [Choice] Geosupport version
+ARG VERSION_GEO="22.2.2"
+FROM nycplanning/docker-geosupport:${VERSION_GEO}
+
+# [Choice] Node.js version: none, lts/*, 16, 14, 12, 10
+ARG NODE_VERSION="none"
+RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
+
+## Install postgres
+RUN apt-get update 
+# && export DEBIAN_FRONTEND=noninteractive \
+RUN apt-get -y install --no-install-recommends postgresql-client 
```

Comment:
follow up on the comment about running `geocode` step in this within the container which I think is a great enhancement. I think right now it is missing the installation for Docker maybe in the line I tagged here for that to work otherwise it is giving me 
```shell
bash/config.sh: line 166: docker: command not found
```